src/state/selection/reducer.ts

```diff
@@ -69,8 +69,8 @@ const actionToConfigMap: TypeToDescriptionMap = {
         ) => {
             return {
                 ...state,
-                visibleAgentKeys: {
-                    ...state.visibleAgentKeys,
+                visibleAgentsNamesAndTags: {
+                    ...state.visibleAgentsNamesAndTags,
```

Comment:
why isn't this called `agentVisibilityMap`?

Reply:
Oh because I missed it! Thanks for catching it, just fixed it